tests/test_serializers.py

```diff
@@ -298,3 +299,49 @@ def test_does_not_allow_object_hook_collision(self):
     def test_does_not_allow_default_collision(self):
         with pytest.raises(pydantic.ValidationError):
             JSONSerializer(dumps_kwargs={"default": "foo"})
+
+
+class TestCompressedSerializer:
```

Comment:
Is it worth having a test that checks that the default compression is in fact `lzma` since it's expected behavior?

Reply:
I'm not too concerned about it since we save the serializer settings alongside results. If the default compression changes, nothing should break. I don't have strong feelings here though, if you want me to add it.